python/eos/tasks.py

```diff
@@ -516,10 +516,10 @@ def predict_observables(analysis_file:str, posterior:str, prediction:str, base_d
 
 
 # Run analysis steps
-@task('run', '', modules=['networkx'])
-def run(analysis_file:str, base_directory:str='./', dry_run:bool=False, executor:str='serial'):
+@task('run', '')
+def run(analysis_file:str, id:str, base_directory:str='./', dry_run:bool=False):
     """
-    Runs a list of predefined steps recorded in the analysis file.
+    Runs one out of a list of predefined steps recorded in the analysis file.
```

Comment:
The comment should then be `#Run one analysis step` in line 518.
Should we also add a `run-all` task?

Reply:
> The comment should then be `#Run one analysis step` in line 518.
Yes!

> Should we also add a `run-all` task?
No, since we might get the ordering (dependencies)  wrong!

Reply:
Done.